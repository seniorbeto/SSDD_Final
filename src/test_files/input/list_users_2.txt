register beto
list_users
quit