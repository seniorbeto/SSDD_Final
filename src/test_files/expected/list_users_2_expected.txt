c> c> REGISTER OK
c> c> LIST_USERS FAIL, USER NOT CONNECTED
c>
+++ FINISHED +++